.travis.yml

```diff
@@ -25,9 +25,9 @@ env:
 matrix:
   fast_finish: true
   include:
-    # Run PHPCS against WPCS. I just picked to run it against 5.5.
-    - php: 5.5
-      env: PHPCS_BRANCH=2.9 SNIFF=1
+    # Run PHPCS against WPCS. I just picked to run it against 7.0.
```

Comment:
Higher PHPCS branch: benefit from the latest fixes upstream
Higher PHP version: benefit from more tokens being natively recognized (as still not everything is backfilled in PHPCS).